l10n_it_fatturapa_in/views/account_view.xml

```diff
@@ -139,6 +140,9 @@
                     <field name="virtual_stamp" readonly="1"></field>
                     <field name="stamp_amount" readonly="1"></field>
                 </field>
+                <field name="invoice_line" position="attributes">
+                    <attribute name="context">{'form_view_ref' : 'l10n_it_fatturapa_in.view_invoice_line_form_fatturapa_in'}</attribute>
```

Comment:
ho diviso i due form, perche' in fattura PA in deve essere tutto readonly , mentre in fattura PA out devono essere scrivibili quei campi..